test/lit.site.cfg.in

```diff
@@ -76,6 +74,12 @@ if "@CMAKE_GENERATOR@" == "Xcode":
     config.environment['PATH'] = \
       os.path.pathsep.join((xcode_bin_dir, config.environment['PATH']))
 
+if "@LLDB_ENABLE@" == "TRUE":
+    config.available_features.add('lldb')
+    for root, dirs, files in os.walk("@LLDB_BUILD_DIR@"):
+        if root.endswith("site-packages"):
+            config.substitutions.append(('%lldb-python-path', root))
```

Comment:
Ah, indeed, test now attached. Good catch, thanks!

Re: break, will add that. In practice it doesn't matter (the build dir for lldb is shallow and contains exactly one match).

Re: {} in substitutions: {} don't seem to be used in current lit substitutions, see [lit.cfg](https://github.com/apple/swift/blob/master/test/lit.cfg#L337).